src/System.ServiceModel.Primitives/ref/project.lock.json

```diff
@@ -2209,21 +2209,21 @@
         "lib/net46/System.Security.Cryptography.Primitives.dll",
         "lib/xamarinios10/_._",
         "lib/xamarinmac20/_._",
-        "ref/dotnet5.4/System.Security.Cryptography.Primitives.dll",
+        "ref/dotnet5.1/System.Security.Cryptography.Primitives.dll",
```

Comment:
@ericstj, can you comment on whether a generations rollback like this is acceptable?